### PR TITLE
Set a PR label / comment if CI passes/fails

### LINKED ISSUE #2: Notify Contributor / label PR if checks fail
GitHub API allows us to check if a CI job failed (the new travis check uses a different API than the older travis webhook integration). PRs with the failed job need the label `tests-fail`. The label needs to be removed when the CI passes again.

It's probably a good idea to also put a comment into the PR because people don't get emails when a label is added/removed. The text for the comment should come from generic config file (we need to keep in mind that other orgs might want to roll this out too).

### PR DESCRIPTION
**Pull request by bastelfreak**
This fixes #2

### CODE REVIEW

**Comment by bastelfreak** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ app/models/pull_request.rb  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/VoxPupuli/vox-pupuli-tasks/pullRequest?prid=5444021&bid=17851631)

**Comment by Flipez** [comment]
This will fix https://github.com/voxpupuli/vox-pupuli-tasks/issues/2

**Comment by Flipez** [comment]
Please only merge if there is time to validate the functionality afterwards

File: app/models/pull_request.rb [hunk 1]
```diff
@@ -159,13 +183,30 @@ def validate(saved_changes)
...
-  def queue_validation
-    ValidatePullRequestWorker.perform_async(id, saved_changes) if saved_changes? || mergeable.nil?
+  def validate_status
+    label = Label.tests_fail
+    # get current status
+    state = begin
+              statuses = Github.client.combined_status(gh_repository_id, github[:head][:sha])
+              statuses[:state]
+            rescue StandardError
+              nil
```
Comment (ekohl) [inline app/models/pull_request.rb:193]: Maybe you want to capture and log the error here and then return. That's probably easier to debug that `state: nil` later on.

**Comment by bastelfreak** [comment]
This is currently blocked by https://github.com/voxpupuli/vox-pupuli-tasks/issues/159

**Comment by bastelfreak** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ app/models/pull_request.rb  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/VoxPupuli/vox-pupuli-tasks/pullRequest?prid=5444021&bid=17851631)

**Comment by bastelfreak** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ app/models/pull_request.rb  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/VoxPupuli/vox-pupuli-tasks/pullRequest?prid=5444021&bid=17851631)

**Pull request merged**